eureka-core/src/test/java/com/netflix/eureka/GzipEncodingEnforcingFilterTest.java

```diff
@@ -89,7 +89,6 @@ public void testForceGzip() throws Exception {
     @Test
     public void testForceGzipOtherHeader() throws Exception {
         noneGzipRequest();
-        when(request.getHeader("Test")).thenReturn("ok");
```

Comment:
What changed here?

Reply:
Updating the Mockito version to 3.4.0 changes the strictness of mock checks. That call `request.GetHeader("Test")` was never called during the test so it fails with `unnecessarystubbingexception`. It's not necessary for the test so I removed it.